### DISCUSSION TITLE
Update Kotaemon in HuggingFace Space

### DISCUSSION THREAD

**Discussion by 3-ark** [Cinnamon/kotaemon#573] [Q&A]
Hi Kotaemon Team,

You guys are doing great work everyday, I can see that long changlog, so I am afraid to miss those new features in newer versions. Currently I deployed kotaemon in huggingface space. I don't know if it's up-to-date. If yes, that's great, if not, can you guys please update that part [here](https://cinnamon.github.io/kotaemon/online_install/) too?

Thanks

**Comment by taprosoft** [comment]
@3-ark thanks for the interest. We will update the tutorial a bit on how to update HF space installation.
Basically, you just need to go to Space settings -> Factory Rebuild to update latest Docker image. However, keep in mind that the data is not persisted between updates.

![image](https://github.com/user-attachments/assets/6f87eba2-f5cd-40ae-b5b3-c6a0f0d9603f)

![image](https://github.com/user-attachments/assets/d592b21c-5a58-446c-bdfa-8b7bccda62d1)
